File: freecodecamp/daily-code-challenge/2026-02-20.py
def is_valid_trick(trick_name):
    valid_first = ["Misty", "Ghost", "Thunder", "Solar", "Sky", "Phantom", "Frozen", "Polar"]
    valid_second = ["Twister", "Icequake", "Avalanche", "Vortex", "Snowstorm", "Frostbite", "Blizzard", "Shadow"]
    
    first, second = trick_name.split(" ")
    return first in valid_first and second in valid_second

File: freecodecamp/daily-code-challenge/2026-02-20.js
function isValidTrick(trickName) {
  const validFirst = ["Misty", "Ghost", "Thunder", "Solar", "Sky", "Phantom", "Frozen", "Polar"];
  const validSecond = ["Twister", "Icequake", "Avalanche", "Vortex", "Snowstorm", "Frostbite", "Blizzard", "Shadow"];
  
  const [first, second] = trickName.split(" ");
  return validFirst.includes(first) && validSecond.includes(second);
}
